queries/r/textobjects.scm

```diff
@@ -53,9 +53,64 @@
 
 ; parameter
 
-(formal_parameters
-  (identifier) @parameter.inner) @parameter.outer
+((formal_parameters
+  . [
+      (identifier)
+      (float)
+      (integer)
+      (complex)
+      (true)
+      (false)
+      (string)
+      (call)
+      (namespace_get)
+      (namespace_get_internal)
+      (default_parameter)
+      (dots)
+      (na)
+      (nan)
+      (null)
+      (inf)
+      (binary)
+      (unary)
+      (pipe)
+      (subset)
+      (subset2)
+      (dollar)
+      (slot)
+    ] @parameter.inner
+    . ","? @_end
+  )
+  (#make-range! "parameter.outer" @parameter.inner @_end))
+
 
 ((arguments
-  (identifier) @parameter.inner . ","? @_end)
+  . [
+      (identifier)
+      (float)
+      (integer)
+      (complex)
+      (true)
+      (false)
+      (string)
+      (call)
+      (namespace_get)
+      (namespace_get_internal)
+      (default_argument)
+      (dots)
+      (na)
+      (nan)
+      (null)
+      (inf)
+      (binary)
+      (unary)
+      (pipe)
+      (function_definition)
+      (subset)
+      (subset2)
+      (dollar)
+      (slot)
+    ] @parameter.inner
```

Comment:
Couldn't we just use `(_)`. Otherwise we might forget something or need to update when a node name changes.